feat: adding recipes tracker workshop (#56456)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/workshop-recipe-tracker/66fbcdfed6848e48947dbbc9.md

---
id: 66fbcdfed6848e48947dbbc9
title: Step 1
challengeType: 1
dashedName: step-1
---

# --description--

> In this project, you'll create a recipe tracker using JavaScript objects.

Reviewer: ```suggestion
In this workshop, you will create a recipe tracker using JavaScript objects.
```

Start by creating an empty array named `recipes`. This is the array you'll push the recipe objects into later.

# --hints--

> You should create an array named `recipes`

Reviewer: ```suggestion
You should create an array named `recipes`.
```
